### ISSUE TITLE
onvif traceback

### ISSUE THREAD

**Issue by adrianoden** [JimStar/reolink_cctv#73]
**Describe the bug**
Host X.X.X.X:80 failed mapping JSON data: 'onvifEnable', traceback: Traceback (most recent call last): File "/usr/local/lib/python3.10/site-packages/reolink_ip/api.py", line 1186, in map_host_json_response self._onvif_enabled = net_port["onvifEnable"] == 1 KeyError: 'onvifEnable'

**To Reproduce**
Located in logs, not sure of trigger

**Debug log would be very helpful**
If applicable, switch on debug-logging, and attach here a full log-file: from the HA restart, and covering the time where the issue happened.  
```
logger:
  default: warning
  logs:
    custom_components.reolink_cctv: debug
    reolink_ip: debug
```

**Screenshots**

**Environment:**
Please provide useful information about your environment, like:
 - Home Assistant version
 - Reolink NVR/camera model
 - NVR/camera firmware

**Additional context**
Maybe any other context about the problem here.

**Comment by JimStar** [comment]
What camera/NVR?

**Comment by JimStar** [comment]
Must be fixed in [0.0.34](https://github.com/JimStar/reolink_cctv/releases/tag/0.0.34).
